recipe/meta.yaml

```
@@ -1,45 +1,59 @@
 {% set name = "cuda-python" %}
-{% set version = "11.8.1" %}
+{% set version = "12.0.0" %}
 {% set major_version = version.split(".")[0] %}
 {% set minor_version = version.split(".")[1] %}
 {% set patch_version = version.split(".")[2] %}
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: ef54ada5d607387f59950cfb3c3852e5816b62824402d289227b8066e8353489
+  sha256: 7df0f84c4b6210112353374a7d992bfd5bba9c97358b39ea03b61bd634ce9c8a
 
 build:
-  number: 2
-  skip: true  # [py<37 or osx or cuda_compiler_version != "11.8"]
+  number: 0
+  skip: true  # [win or osx or cuda_compiler_version != "12.0"]
```

Since `aarch64` & `ppc64le` are currently not yet supported on CUDA 12, should we skip them?

```suggestion
  skip: true  # [aarch64 or ppc64le or win or osx or cuda_compiler_version != "12.0"]
```